### PR TITLE
venv: Log pip version on venv creation, clarify empty output logging

### PR DESCRIPTION
**Pull request by carlosperate**
I find myself asking a lot of users with Crash Reports to run commands in the terminal to determine the pip version running inside the virtual environment, so this PR logs that to the venv creation process.

It also logs "No output received." whenever there is an empty output from venv creation/verification commands, as otherwise we end up with empty entries in the log, which are a bit hard to follow.

### CODE REVIEW

**Review by tjguk — APPROVED** [review]
Looks fine on paper. Haven't cloned/run but I assume you have :)

**Comment by carlosperate** [comment]
Yes, I've run it in a few different scenarios to make sure it didn't interfeer with anything. Thanks for the review! I'll merge 👍
